presto-docs/src/main/sphinx/release/release-0.288.rst

```diff
@@ -0,0 +1,91 @@
+* Add support for non default keystore and truststore type in presto CLI and JDBC. :pr:`22556`
+* Add support for querying system.runtime.tasks table in Presto C++ clusters. :pr:`21416`
+* Add two system configuration properties to specify the reserved query memory capacity on Presto C++ clusters: ``query-reserved-memory-gb`` is the total amount of memory in GB reserved for the queries on a worker node. ``memory-pool-reserved-capacity`` is the amount of memory in bytes reserved for each query. :pr:`22593`
+* Replace the Presto native stats definition and reporting for the memory allocator, in-memory cache and ssd cache metrics from Presto repo to Velox repo, with the metrics names changing from presto_cpp.<metrics_name> to velox.<metrics_name>. :pr:`22751`
+* Remove deprecated feature and configuration property ``deprecated.group-by-uses-equal``, which allowed group by to use equal to rather than distinct semantics. :pr:`22888`
+* Upgrade CI pipeline to build and publish Presto C++ worker docker image. :pr:`22806`
+* Upgrade Alluxio to 313. :pr:`22958`
+* Upgrade io.jsonwebtoken artifacts to 0.11.5. :pr:`22762`
+* Upgrade fasterxml.jackson artifacts to 2.11. :pr:`22417`
+
+Hive Connector Changes
+______________________
+* Fix hash calculation for Timestamp column to be hive compatible when writing to a table bucketed by Timestamp. :pr:`22980`
+* Improve affinity scheduling granularity from a file to a section of a file by adding a ``hive.affinity-scheduling-file-section-size`` configuration property and ``affinity_scheduling_file_section_size`` session property. The default file size is 256MB. :pr:`22563`
+* Add AWS Security Mapping to allow flexible mapping of Presto Users to AWS Credentials or IAM Roles for different AWS Services. :pr:`21622`
+* Add config property ``hive.legacy-timestamp-bucketing`` and session property ``hive.legacy_timestamp_bucketing`` to use the original hash function for Timestamp column, which is not hive compatible. :pr:`22980`
+* Add support for ``NOT NULL`` column constraints in the CREATE TABLE and ALTER TABLE statements for the Hive connector. :pr:`22064`
+
+Iceberg Connector Changes
+_________________________
+* Improve the partition specs that must be checked to determine if the partition supports metadata deletion or predicate thoroughly pushdown. :pr:`22753`
+* Improve time travel ``TIMESTAMP (SYSTEM_TIME)`` syntax to include timestamp-with-time-zone data type. :pr:`22851`
+* Improve time travel ``VERSION (SYSTEM_VERSION)`` syntax to include snapshot id using `BIGINT` data type. :pr:`22851`
+* Add procedure `expire_snapshots` to remove old snapshots in Iceberg. :pr:`22609`
+* Add support for Iceberg REST catalog. :pr:`22417`
+* Add time travel ``BEFORE`` syntax for Iceberg tables to return historical data. :pr:`22851`
+* Add support for metadata delete with predicate on non-identity partition columns when they align with partitioning boundaries. :pr:`22554`
```

Comment:
```suggestion
* Add support for metadata delete with predicate on non-identity partition columns when they align with partitioning boundaries. :pr:`22554`
* Add configuration of Iceberg split manager threads using the iceberg.split-manager-threads configuration property. :pr:`22754`
```
Moved `* Add configuration of Iceberg split manager threads using the iceberg.split-manager-threads configuration property. :pr:`22754` ` to group Adds together before the Replace entry in Iceberg Connector Changes, following the sequence in the [Order of Changes](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes) in the Release Notes Guidelines.